app/commands.js

```diff
@@ -3,96 +3,131 @@ import staticResponses from './data/staticResponses.js';
-    const peopleString = nameArray.join('\n');
-    ctx.reply(peopleString)
-  });
+const getRandomNum = (limit) => {
+  return Math.floor(Math.random() * limit);
+}
 
-  app.command('qtvv', ctx => {
-    ctx.reply(staticResponses['qtvv']);
-  });
+const getSearchGiphyURL = (queryString) => {
+  return `${giphyUrl}search?api_key=${giphyToken}&q=${encodeURIComponent(queryString)}&limit=${GIPHY_API_LIMIT}`;
+}
 
-  app.command('buenas', ctx => {
-    const index = getRandomNum(staticResponses.buenas.length);
-    ctx.reply(staticResponses.buenas[index]);
+export const rules = (ctx) => {
+  let rules = cache.getValue('allRules');
+  let i = 1;
+  rules = rules.map(rule => {
+    return `${i++}.- ${rule}`;
   });
+  const ruleString = rules.join('\n');
+  ctx.reply(ruleString)
+};
 
-  app.command('next', ctx => {
-    const name = getNextName(people);
-    setCurrent();
-  });
-  
-  app.command('0fucks', ctx => {
-    const queryString = 'zero fucks';
-    return promiseReply(queryString, ctx);
-  });
+export const mortos = (ctx) => {
+  const people = cache.getValue('people');
+  const nameArray = people.map(person => person.current ? `*${person.name}*` : person.name);
+  const peopleString = nameArray.join('\n');
+  ctx.reply(peopleString)
+};
 
-  app.command('fucku', ctx => {
-    const queryString = 'fuck you';
-    return promiseReply(queryString, ctx);
-  });
+export const qtvv = (ctx) => {
+  ctx.reply(staticResponses['qtvv']);
+};
 
-  const promiseReply = (queryString, ctx) => {
-    const random = getRandomNum(GIPHY_API_LIMIT);
-    const url = getSearchGiphyURL(queryString);
-    axios.get(url)
-      .then(response => {
-        if(response.status === httpStatus.OK) {
-          ctx.reply(response.data.data[random].url);
-        }
-      })
-      .catch(error => {
-        console.log(error);
-        ctx.reply('error');
-      });
-  }
+export const buenas = (ctx) => {
+  const index = getRandomNum(staticResponses.buenas.length);
+  ctx.reply(staticResponses.buenas[index]);
+};
 
-  const getRandomNum = (limit) => {
-    return Math.floor(Math.random() * limit);
+const getNextName = (index) => {
+  let name = '';
+  const people = cache.getValue('people');
+  const lastIndex = people.length - 1;
+  if (index === 0) {
+    name = people[lastIndex].name;
+  } else {
+    name = people[index - 1].name;
   }
 
-  const getSearchGiphyURL = (queryString) => {
-    return `${giphyUrl}search?api_key=${giphyToken}&q=${encodeURIComponent(queryString)}&limit=${GIPHY_API_LIMIT}`;
-  }
+  return name;
+}
 
-  app.command('jaja', ctx => {
-    ctx.reply(staticResponses['jaja']);
-  });
-  
-  app.command('chupala', ctx => {
-    const index = getRandomNum(staticResponses.chupala.length);
-    ctx.reply(staticResponses.chupala[index]);
+export const next = (ctx) => {
+  const people = cache.getValue('people');
+  findCurrentMorto().then((morto) => {
+    removeCurrent(morto.name);
+    return morto.name;
+  }).then((result) => {
```

Comment:
instead of `(result)` you could rename to `(mortoName)` and avoid extra variables.